timely_beliefs/visualization/ridgeline.py

```diff
@@ -0,0 +1,233 @@
+    raise ValueError("No mean cp value")
+
+
+def ridgeline_plot(start_time, df, start=0, end=168, fixedviewpoint = False):
+    """
+    Creates ridgeline plot by selecting a belief history about a specific event
+
+    @param start_time : datetime string of selected event
+    @param df : timely_beliefs DataFrame
+    @param start : start of hours before event time
+    @param end : end of hours before event time
+    @param fixedviewpoint : if true create fixed viewpoint plot
+    """
+    #set params
+    if fixedviewpoint == True:
+        start = 0
+    #out of bounds checks
+    if end < 0 or end > 168:
+        raise ValueError("End of the forecast horizon must be between 0 and 168 hours.")
+    if start < 0 or start > end:
+        raise ValueError("Start of the forecast horizon must be between 0 and 168 hours.")
+    #to include last observation
+    end += 1
+    #get cps
+    cp_list, pred_temp_05, pred_temp_0, pred_temp_2 = create_cp_data(df,start,end,start_time,fixedviewpoint)
+    #make means and std
+    mean = np.array([float(i) for i in pred_temp_05])
+    std1 = np.array([(float(pred_temp_0[i])-float(pred_temp_05[i]))/(np.sqrt(2)*erfinv((2*cp_list[0])-1)) for i in range(len(pred_temp_05))])
+    std2 = np.array([(float(pred_temp_2[i])-float(pred_temp_05[i]))/(np.sqrt(2)*erfinv((2*cp_list[1])-1)) for i in range(len(pred_temp_05))])
+    std = (std1+std2)/2
+    #plot everything
+    show_plot(mean, std, start, end, fixedviewpoint)
+
+
+def show_plot(mean, std, start, end, fixedviewpoint=False):
+    """
+    Creates and shows ridgeline plot
+    @param mean: list of float, mean values
+    @param std: list of float, standard deviation values
+    @param start: int,start hours before event-time
+    @param end: int,end hours before event-time
+    @param fixedviewpoint : BOOLEAN, if true create fixed viewpoint plot
+    """
+    #amount of lines to draw
+    nr_lines = len(mean)
+    x = np.linspace(-10, 30, 500)
+    frame = pd.DataFrame()
+    #generate points for each line
+    for i in range(nr_lines):
+        frame["{}".format(i)] = stats.norm.pdf(x, mean[i], std[i])
+    #set color pallete to viridis
+    pallete = viridis(nr_lines)
+    #set list reversed or not depending on viewpoint
+    if fixedviewpoint:
+        cats = list(frame.keys())
+    else:
+        cats = list(reversed(frame.keys()))
+    source = ColumnDataSource(data=dict(x=x))
+    #creat figure
+    p = figure(y_range=cats, plot_width=900, x_range=(-5, 30), toolbar_location=None)
+    #make a ridge line in the figure
+    for i, cat in enumerate(reversed(cats)):
+        y = ridge(cat, frame[cat], 50)
+        source.add(y, cat)
+        p.patch('x', cat, alpha=0.6, color=pallete[i], line_color="black", source=source)
+    #added y axis with the right ticks etc depending on fixedviewpoint
+    if fixedviewpoint:
+        p.yaxis.axis_label = 'Number of hours after event_start'
+        y_ticks = list(np.arange(0, nr_lines, 5))
```

Comment:
Consider drawing your ticks at every 6 hours instead of every 5 hours, so the viewer can more easily identify horizons of interest (like 24, 48 and 72 hours). This would entail changing a 5 to a 6 in three other lines, too.